### ISSUE TITLE
Architecture Committee: Increase the amount of seats from 5 to 7

### ISSUE THREAD

**Issue by fidencio** [kata-containers/community#312]
Folks, regardless of the results of the current elections, it's been made clear that Kata Containers is a growing community and it's still very appealing for several individuals and companies.

This last election round we have **6** runners for 3 seats and, regardless of the results, we'll see some good additions for the community not being present in the Architecture Committee.

Considering this, and also the growing interest on different areas (peer pods, confidential containers, and Kata Containers itself ...) I'd like to propose that for the next election round (the one that happens at the beginning of the 2023) we increase the amount of seats **from 2 to 4**, keeping the balance of the members switching between elections and not causing a big disruption.

I'd like to hear here from the current A/C members, from the newly elected A/C members, and from the community.

**Comment by bergwolf** [comment]
It is great to see that there is growing interest in the Kata community and the architecture committee. Now more and more companies are adopting Kata Containers in production. I would like to say that we should be open and invite more folks to the community including the architecture committee.

So a big plus from my side for increasing the A/C member seats from 2 to 4 in the spring election. Thank you for raising it @fidencio!

**Comment by sameo** [comment]
I fully agree here. I also think the AC inclusion process should also be more flexible, and maybe find some other solutions than an election? This round's candidate all want to contribute to the project directions, and would all be excellent AC members. I'd prefer that we welcome most of these initiatives rather than having to rule some of them out through elections.

**Comment by egernst** [comment]
SGTM

**Comment by amshinde** [comment]
I am all for increasing the A/C member seats given the growing interest in Kata and related efforts,  and the great list of candidates we have for this round. I am open to increasing the amount of seats for this round itself, increasing it to 4 given the number of candidates we have for this election.  If there are concerns around deadlock in decision making, I think we should not be too concerned about it, since we havent come across such a situation in the past. 
If others strongly feel its better to push this to next election, I am fine with it as long as we add the seats at some point.

**Comment by fidencio** [comment]
> I am all for increasing the A/C member seats given the growing interest in Kata and related efforts, and the great list of candidates we have for this round. I am open to increasing the amount of seats for this round itself, increasing it to 4 given the number of candidates we have for this election. If there are concerns around deadlock in decision making, I think we should not be too concerned about it, since we havent come across such a situation in the past. If others strongly feel its better to push this to next election, I am fine with it as long as we add the seats at some point.

I'm personally against having the amount of seats increased **at this election**, as if we keep following elections we'd end up with two very disbalanced rounds (2 seats vs 5 seats, for instance).

Having this round planned as it was, and then increasing the next one to 4 sounds a little bit more ideal (at least to me), and doesn't change the rules of the election in the middle of the running process.

I also would like to hear about this from the newly elected committee members, which we may get to know Tomorrow, and even consider the suggestion raised by @sameo (checking with OIF first, of course), of having the A/C as something less bureaucratic than elections.

**Comment by jiangliu** [comment]
I'm all in for increasing the AC seats, a bigger community is a good thing for KATA.
